G1ANT.Addon.UI/G1ANT.Addon.UI/Api/UIElement.cs

```diff
@@ -92,53 +93,35 @@ public WPathStructure ToWPath(UIElement root = null)
 
         public void Click()
         {
-            object pattern;
-            if (automationElement.TryGetCurrentPattern(InvokePattern.Pattern, out pattern))
+            if (automationElement.TryGetClickablePoint(out var pt))
```

Comment:
I think clicking with mouse should be the last try after Invoke or Selection patterns are not supported. What do you think?

Reply:
Win32 mouse click is more accurate in this method (Click()) on UI Element especially in Win32 app, patterns like InvokePatterns works with UI Automation and very often do nothing or throw exception, in this sample on Windows 7